### PR TITLE
Doc: update Flink doc for sink metrics

### PR DESCRIPTION
**Pull request by stevenzwu**

### CODE REVIEW

File: docs/flink-getting-started.md [hunk 1]
```diff
@@ -747,6 +747,44 @@ FlinkSink.builderFor(
...
+| lastCommitDurationMs            | Gague  | The duration (in milli) that the Iceberg table commit takes.               |
+| committedDataFilesCount         | Counter | Number of data files committed.                                            |
+| committedDataFilesRecordCount   | Counter | Number of records contained in the committed data files.                   |
+| committedDataFilesByteCount     | Counter | Number of bytes contained in the committed data files.                     |
+| committedDeleteFilesCount       | Counter | Number of delete files committed.                                          |
+| committedDeleteFilesRecordCount | Counter | Number of records contained in the committed delete files.                 |
+| committedDeleteFilesByteCount   | Counter | Number of bytes contained in the committed delete files.                   |
+| elapsedSecondsSinceLastSuccessfulCommit| Gague  | Elapsed time (in seconds) since last successful Iceberg commit.            |
+
+`elapsedSecondsSinceLastSuccessfulCommit` is an ideal alerting metric for these scenarios.
```
Comment (pvary) [inline docs/flink-getting-started.md:784]: I do not really get what `these scenarios`  refers to in this sentence

File: docs/flink-getting-started.md [hunk 2]
```diff
@@ -747,6 +747,44 @@ FlinkSink.builderFor(
   .append();
 ```
 
+### monitoring metrics
+
+The following Flink metrics are provided by the Flink Iceberg sink.
+
+Parallel writer metrics are added under the sub group of `IcebergStreamWriter`. 
+They should have the following key-value tags.
```
Comment (pvary) [inline docs/flink-getting-started.md:755]: Would it be worthwhile to provide an example set of metrics values in a separate table for both objects?

Comment (stevenzwu) [reply docs/flink-getting-started.md:755]: I am not sure if it is too helpful with a table for two rows of key-values.

File: docs/flink-getting-started.md (hunk 1 above)
Comment (stevenzwu) [reply docs/flink-getting-started.md]: it refers the scenarios described below.  I assume you also mean the descriptions below aren't clear. I updated the description. Please check if it helps.

File: docs/flink-getting-started.md (hunk 2 above)
Comment (pvary) [reply docs/flink-getting-started.md:755]: Do I understand correctly that the metrics key would look like this: `iceberg.my_db.my_table.0.lastFlushDurationMs`

Comment (pvary) [reply docs/flink-getting-started.md:755]: Not sure what `sub group` means. Maybe I just have to look it up

File: docs/flink-getting-started.md [hunk 3]
```diff
@@ -747,6 +747,50 @@ FlinkSink.builderFor(
...
+| lastCommitDurationMs            | Gague  | The duration (in milli) that the Iceberg table commit takes.               |
+| committedDataFilesCount         | Counter | Number of data files committed.                                            |
+| committedDataFilesRecordCount   | Counter | Number of records contained in the committed data files.                   |
+| committedDataFilesByteCount     | Counter | Number of bytes contained in the committed data files.                     |
+| committedDeleteFilesCount       | Counter | Number of delete files committed.                                          |
+| committedDeleteFilesRecordCount | Counter | Number of records contained in the committed delete files.                 |
+| committedDeleteFilesByteCount   | Counter | Number of bytes contained in the committed delete files.                   |
+| elapsedSecondsSinceLastSuccessfulCommit| Gague  | Elapsed time (in seconds) since last successful Iceberg commit.            |
+
+`elapsedSecondsSinceLastSuccessfulCommit` is an ideal alerting metric
```
Comment (pvary) [inline docs/flink-getting-started.md:784]: Did we change the name of this metrics?

File: docs/flink-getting-started.md (hunk 1 above)
Comment (pvary) [reply docs/flink-getting-started.md]: +1

**Review by pvary — APPROVED** [review]

File: docs/flink-getting-started.md (hunk 2 above)
Comment (stevenzwu) [reply docs/flink-getting-started.md:755]: https://nightlies.apache.org/flink/flink-docs-master/docs/ops/metrics/#user-scope

```Java
counter = getRuntimeContext()
  .getMetricGroup()
  .addGroup("MyMetrics")
  .counter("myCounter");

counter = getRuntimeContext()
  .getMetricGroup()
  .addGroup("MyMetricsKey", "MyMetricsValue")
  .counter("myCounter");
```

**Comment by stevenzwu** [comment]
Merged. Thanks @pvary for the review.

**Pull request merged**